src/Common/Commands.Common/Properties/Resources.Designer.cs

```diff
@@ -1207,11 +1208,11 @@ public class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Internal Server Error.
+        ///   Looks up a localized string similar to public Server Error.
```

Comment:
Is this comment change intentional?  (internal server error -> public server error)
Or is this comment changed during replacing internal method to public?